crates/rolldown/tests/fixtures/errors/ambiguous_external_namespace/artifacts.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+assertion_line: 176
+expression: content
+input_file: crates/rolldown/tests/fixtures/errors/ambiguous_external_namespace
+---
+# Errors
+
+## AMBIGUOUS_EXTERNAL_NAMESPACES
+
+```text
+[AMBIGUOUS_EXTERNAL_NAMESPACES] Error: Warning: Found ambiguous export.
```

Comment:
The diagnostic is incorrect, you should adjust `Severity` and remove `Warning: ` in the content.